packages/utils/src/lib/crypto.ts

```diff
@@ -9,27 +9,42 @@ export const NEWM_POLICY_ID = isProd
 export const NEWM_ASSET_NAME = isProd ? "4e45574d" : "744e45574d";
 
 interface FormatCurrencyOptions {
-  readonly includeSymbol?: boolean;
+  readonly includeCurrencySymbol?: boolean;
+  readonly includeEstimateSymbol?: boolean;
   readonly precision?: number;
 }
 
 /**
- * Formats a numerical NEWM amount with the correct decimal
- * places and symbol.
+ * Formats a numerical NEWM amount with three decimal
+ * places and the correct symbol.
  */
 export const formatNewmAmount = (
   amount?: number,
-  options: FormatCurrencyOptions = { includeSymbol: true, precision: 2 }
+  options: FormatCurrencyOptions = {
+    includeCurrencySymbol: true,
+    includeEstimateSymbol: false,
+    precision: 3,
+  }
 ) => {
   if (!amount) return "0 Ɲ";
 
-  const { includeSymbol = true, precision = 2 } = options;
+  const {
+    includeCurrencySymbol = true,
+    includeEstimateSymbol = false,
+    precision = 3,
```

Comment:
Are these deconstruction default values necessary when the params already have default values? Existed before I touched this, but added estimate symbol for deconstruction use and default value of false for consistency.

Reply:
Copilot says they are redundant. So it says, so shall it be. 

I believe I quickly tested it, but I would like to spend a few more minutes to double check, smallish but will create a ticket to review this.